### PR TITLE
Add travis config and deploy file

### PR DESCRIPTION
**Pull request by sallar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "dev": "grunt next && next",
     "build": "grunt next && next build && next export",
     "start": "next start",
+    "test": "echo 'DONE'",
```
Comment (perry-mitchell) [inline package.json]: What about making this `npm run build` so we at least test the build procedure?

File: deploy.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# the private key
+set +x
+
+echo -e $SSH_KEY | base64 --decode > ~/.ssh/id_rsa
+chmod 600 ~/.ssh/id_rsa
+
+set -x
+
+npm run build
+npm run deploy
```
Comment (perry-mitchell) [inline deploy.sh]: Maybe add deletion of the key afterwards?

**Review by perry-mitchell — CHANGES_REQUESTED** [review]

**Comment by sallar** [comment]
@perry-mitchell done

File: deploy.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Turn off command traces while dealing with
+# the private key
+set +x
+
+echo -e $SSH_KEY | base64 --decode > ~/.ssh/id_rsa
+chmod 600 ~/.ssh/id_rsa
+
+npm run deploy
```
Comment (perry-mitchell) [inline deploy.sh]: Sorry to be a burden! I guess we want `deploy` to run outside of the protected traces.. Like:

```shell
set +x

# SSH stuffs

set -x

# Deploy

set +x

# Delete SSH

set -x
```

**Review by perry-mitchell — CHANGES_REQUESTED** [review]

File: deploy.sh (hunk 3 above)
Comment (perry-mitchell) [reply deploy.sh]: Does that make sense? That way our private stuff is hidden but the deploy log is still visible for debugging :)

**Review by perry-mitchell — APPROVED** [review]

**Pull request merged**